be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/MessageData.scala

```diff
@@ -0,0 +1,6 @@
+package ch.epfl.pop.model.network.method.message.data
+
+trait MessageData {
+  val _object: ObjectType.ObjectType
```

Comment:
why put an underscore here?

Reply:
Because object is a reserved keyword in scala. There's no way to name a varialbe object :/